libpod/image/search.go

```diff
@@ -64,11 +64,11 @@ type SearchFilter struct {
 // SearchImages searches images based on term and the specified SearchOptions
 // in all registries.
 func SearchImages(term string, options SearchOptions) ([]SearchResult, error) {
-	// Check if search term has a registry in it
-	registry, err := sysreg.GetRegistry(term)
-	if err != nil {
-		return nil, errors.Wrapf(err, "error getting registry from %q", term)
+	registry := ""
+	if spl := strings.SplitN(term, "/", 2); len(spl) > 1 {
+		registry = spl[0]
 	}
+
 	if registry != "" {
 		term = term[len(registry)+1:]
 	}
```

Comment:
(Absolutely non-blocking: It feel be a tiny bit cleaner to me if this used `spl[1]` in the condition above instead of an independent parsing code — but AFAICS it works exactly the same.)